src/omero_zarr/masks.py

```diff
@@ -55,7 +57,26 @@ def image_masks_to_zarr(image, args):
             for (roi_id, roi) in masks.items():
                 saver.save([roi], str(roi_id))
         else:
-            saver.save(masks.values(), args.mask_name)
+            if args.mask_map:
+
+                mask_map = defaultdict(list)
+                roi_map = {}
+                for (roi_id, roi) in masks.items():
+                    roi_map[roi_id] = roi
+
+                try:
+                    for line in input(args.mask_map):
+                        line = line.strip()
+                        id, name, roi = line.split(",")
+                        mask_map[name].append(roi_map[int(roi)])
```

Comment:
Should probably check if `int(roi)` is found in `roi_map`

Reply:
What behavior would you suggest? I was just going to let it error out as on line 73.

Reply:
OK, fair enough.
Seeing the workflow above, I see that this isn't such a manual (error-prone) workflow, so you're probably OK. At least for now.